src/components/Router/StepComponentMap.js

```diff
@@ -2,15 +2,15 @@ import { h } from 'preact'
 
 import Welcome from '../Welcome'
 import {SelectPoADocument, SelectIdentityDocument} from '../Select'
-import {FrontDocumentCapture, BackDocumentCapture, FaceCapture, LivenessCapture} from '../Capture'
-import {DocumentFrontConfirm, DocumentBackConfirm, FaceConfirm, LivenessConfirm} from '../Confirm'
+import {FrontDocument, BackDocument, Selfie, Video, PoADocument } from '../Capture'
```

Comment:
Yes I had removed it because at some point I renamed the whole group of components. But later on I realised there was no need to rename it. So I agree, they should be brought back